docs/sources/write/style-guide/word-list/index.md

```diff
@@ -144,9 +144,12 @@ The text should always be pre-formatted as inline code (between backticks (\`)).
 
 ## O
 
-### okay
+<!-- vale Grafana.OK = NO -->
+### OK, okay
 
-Use _okay_ rather than _OK_ unless you're referencing the UI.
+Avoid using _OK_ or _okay_ unless you're referencing the UI, because it's too informal.
```

Comment:
```suggestion
Avoid using _OK_ or _okay_ in technical documentation, because it's too informal. The exceptions are when you're referencing or quoting:
- a user interface
- HTTP status codes or other code
```